org.ultimatepp.TheIDE.yml

```diff
@@ -0,0 +1,115 @@
+app-id: org.ultimatepp.TheIDE
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+sdk-extensions: ["org.freedesktop.Sdk.Extension.llvm17"]
+command: run-ide.sh
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --share=ipc
+  - --filesystem=home
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
this is not allowed by default.

Reply:
I just noticed that VS Code is using this permission. TheIDE is integrated development environment and without it, it simply won't work. TheIDE is exactly in the same category as VS Code is.

What is your recommendation here?

Reply:
https://docs.flathub.org/docs/for-app-authors/linter#exceptions